site/app/libraries/SelectWidgets.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace app\libraries;
+
+use app\libraries\Core;
+
+/**
+ * Class SelectWidgets
+ *
+ * Adds the necessary CSS and Js to use Select2 widgets.
+ */
+class SelectWidgets {
+    /**
+     * Adds the necessary CSS and JS to use a Select2 widget.
+     * Select2 widgets allow you to preselect from options as well as
+     * type your own option.
+     *
+     * @param Core $core The core of the Submitty application we will add CSS and Js to.
+     * @return void
+     */
+    public static function addSelect2WidgetCSSAndJs(Core $core): void {
```

Comment:
Instead of having a static function that requires passing `Core` to it, how about just putting this function into the `Output` class, given that it's highly coupled to how that functions.